server/db/MongoDB.go

```diff
@@ -54,13 +54,25 @@ func (db *mongoDB) HasGameByID(id string) bool {
 }
 
 // CreateGame a game with the given ID. Perhaps this should instead just return an id?
-func (db *mongoDB) CreateGame() (*model.Game, error) {
-	myGame := model.Game{Password: ""}
+func (db *mongoDB) CreateGame(gameName string, creatorID string) (*model.Game, error) {
+	player, err := db.LookupPlayer(creatorID)
```

Comment:
This will also throw an error, because you instantiate `err` on line 58 here, where it was being instantiated beforehand on line 70.

Reply:
See my comment above.

Reply:
That's very odd to me... as I've had the same issue prior. but as you said it may have been because I instantiated both variables prior to the command. Everything else looks fine and I can run through a game. Approving now.